tests/functional/modules/test_zos_job_query_func.py

```diff
@@ -123,3 +123,4 @@ def test_zos_job_id_query_short_ids_with_wilcard_func(ansible_zos_module):
     qresults = hosts.all.zos_job_query(job_id="STC00*")
     for qresult in qresults.contacted.values():
         assert qresult.get("jobs") is not None
+        assert qresult.get("jobs")[0].get("job_type").strip() == "STC"
```

Comment:
if this was stripped, how did the `job_type` output looks like ?

` STC ` or similar? is this something we need to take zoau to enhance ?

Reply:
When its value is STC, ZOAU returns `STC ` (with a trailing whitespace). Yeah, probably something worth discussing with ZOAU.

Reply:
Doesn't happen when the value is `JOB`.